docker/docker-compose.yml

```diff
@@ -33,6 +33,20 @@ services:
       POSTGRES_PASSWORD: root
     command: -c 'max_connections=512' -c 'wal_level=logical'
 
+  # POSTGRESQL in-app notification manager
+  in-app-notification-manager-sql:
+    volumes:
+      - ./in-app-notification-manager-db:/docker-entrypoint-initdb.d
+    image: postgres:14
+    ports:
+      - 6004:5433
+    environment:
+      PGPORT: 5433
+      POSTGRES_DB: root
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: root
+    command: -c 'max_connections=512' -c 'wal_level=logical'
```

Comment:
`-c 'wal_level=logical'` is probably not needed since Debezium won't listen to this DB... but I see it was kept also for the read model, no issue to keep it here too